starboard/build/config/modular/variables.gni

```diff
@@ -15,6 +15,6 @@
 assert(current_toolchain == default_toolchain,
        "Cannot access variables for non-default toolchains")
 
-if (!is_host_win) {
-  is_clang_16 = true
+if (is_host_win) {
+  is_clang_16 = false
```

Comment:
Huh, i would have expected we don't need this. Seems like we need to untangle some conditionals elsewhere ?

Reply:
ps5 is on clang15 : https://b.corp.google.com/issues/246412559#comment82

maybe we can revisit this once ps5 is updated to clang16